### ISSUE TITLE
Environment

### ISSUE THREAD

**Issue by fireceW** [Len-Li/Adv3D#1]
Hello, Could you provide the specific version numbers for "mvcc", "mmdet", and "mmdet3d"? I encountered some issues while configuring the environment, thanks.

**Comment by Ustinian-J** [comment]
Yes. I encountered the same question. I wonder whether you have solved it yet.
If you can successfully run the code, can you tell me the specific version?
Thanks very much!

**Comment by Len-Li** [comment]
Thanks for reaching out.

I am using BEVDet dev2.0 version. Basically, it uses mmdet==2.25.1 and mmsegmentation==0.25.0.
You can find detail configuration in [BEVDet/docker/Dockerfile at dev2.0 · HuangJunJie2017/BEVDet (github.com)](https://github.com/HuangJunJie2017/BEVDet/blob/dev2.0/docker/Dockerfile)

**Comment by Ustinian-J** [comment]
Thank you very much. I have just successfully runed the model because of  too much time spent on the dataset.
It's good news to hear your success.
May you good luck.
